test/e2e/tests/test_subscription.py

```
@@ -86,6 +96,7 @@ def _create_sa_token(sa_name, namespace=None, duration="10m"):
         capture_output=True, text=True,
     )
     token = result.stdout.strip()
+    token = ""
     if not token:
         raise RuntimeError(f"Could not create token for SA {sa_name}: {result.stderr}")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)